lib/resources/readme.md

```diff
@@ -11,9 +11,7 @@ Included languages:
 * javascript
 * json
 * markdown
-* objectivec
```

Comment:
Intentional? My understanding is that we had these so Flutter code snippets show up correctly...

Reply:
I assumed this was unused. 

As per https://github.com/dart-lang/dartdoc/issues/1963, it would be nice if we had some automation / process around the upgrade.

Reply:
Updated to include obj-c and kotlin.